forgettable-frontend/src/colours.css

```diff
@@ -0,0 +1,20 @@
+:root {
+    --prmry: #F3D58A; /* main accent */
+    --hilit: #FFF7E3; /* selected cards */
+
+    --lbkgd: #FFFFFF; /* background */
+    --licon: #4D4D4D; /* navbar icons */

```

Comment:
Just curious if there is a way we can do this where it goes background.light or background(light)  or something so that we can just pass in the theme type instead of making if statements all around the code. would make it more expandable for other theme colours in the future too right? correct me if i dont have the right idea for how we are using these colours though haha

Reply:
Good point, I've added separate light/dark theme files as well :)